.github/workflow/pylint.yml

```
@@ -0,0 +1,17 @@
+on: pull_request
+
+jobs:
+  example:
+    runs-on: ubuntu-latest
+    name: Executes pylint
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Lint
+        uses: gabriel-milan/action-pylint@v1
+        with:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}} # GitHub token
+          path: "**.py" # Glob pattern for files to lint
```

It looked into it and apparently this is the pattern that pylint can use for global files. 

```suggestion
          path: "**/*.py" # Glob pattern for files to lint
```